src/Acuminator/Acuminator.Analyzers/FixProviders/DAC/DacPropertyAttributes/DacExtensionDefaultAttributeFix.cs

```diff
@@ -0,0 +1,138 @@
+				CodeAction codeActionUnbound =
+					CodeAction.Create(codeActionNameUnbound,
+										cToken => ReplaceIncorrectDefaultAttribute(context.Document, context.Span, diagnostic.IsBoundField(), cToken),
+										equivalenceKey: codeActionNameUnbound);
+
+				context.RegisterCodeFix(codeActionUnbound, context.Diagnostics);
+			}
+		}
+
+		private async Task<Document> ReplaceIncorrectDefaultAttribute(Document document, TextSpan span, bool isBoundField,CancellationToken cancellationToken)
+		{
+			SyntaxNode root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+
+			 if (!(root?.FindNode(span) is AttributeSyntax attributeNode))
+				return document;
+
+			if (!(attributeNode.Parent is AttributeListSyntax attributeList))
+				return document;
+
+			cancellationToken.ThrowIfCancellationRequested();
+
+			SyntaxGenerator generator = SyntaxGenerator.GetGenerator(document);
+
+			var memberAccessExpression = generator.MemberAccessExpression(generator.IdentifierName(_PXPersistingCheck),
+																		  generator.IdentifierName(_PersistingCheckNothing));
+			var persistingAttributeArgument = generator.AttributeArgument(_PersistingCheck, 
+																		  memberAccessExpression) as AttributeArgumentSyntax;
+			var pxUnboundDefaultAttribute = generator.Attribute(_PXUnboundDefaultAttributeName) as AttributeListSyntax;
+			
+			SyntaxNode modifiedRoot;
+
+			if (isBoundField)
+			{
+				if (attributeNode.ArgumentList != null)
+				{
+					var persistingCheckArguments = from arguments in attributeNode.ArgumentList.Arguments
+													where arguments.GetText().ToString().Contains(_PersistingCheck)
+													select arguments;
+					
+					if (persistingCheckArguments.Any())
+					{
+						AttributeArgumentSyntax argument = persistingCheckArguments.First();
+						persistingAttributeArgument = argument.ReplaceNode(argument.Expression, memberAccessExpression);
+						var newAttributeNode = attributeNode.ReplaceNode(argument, persistingAttributeArgument);
+						var newAttributeList = attributeList.ReplaceNode(attributeNode, newAttributeNode);
+						modifiedRoot = root.ReplaceNode(attributeList, newAttributeList);
+					}
+					else
+					{
+						var newAttributeList = generator.AddAttributeArguments(attributeNode, new SyntaxNode[] { persistingAttributeArgument }) as AttributeListSyntax;
+						modifiedRoot = root.ReplaceNode(attributeNode, newAttributeList.Attributes[0]);
+					}
+				}
+				else
+				{
+					AttributeListSyntax newAttribute = generator.InsertAttributeArguments(attributeNode, 1, new SyntaxNode[] { persistingAttributeArgument }) as AttributeListSyntax;
+					modifiedRoot = root.ReplaceNode(attributeNode, newAttribute.Attributes[0]);
+				}
+			}
+			else
+			{
+				modifiedRoot = root.ReplaceNode(attributeNode, pxUnboundDefaultAttribute.Attributes[0]);
+			}
+
+			return document.WithSyntaxRoot(modifiedRoot);
+		}
+
+
+	}
+
+	internal static class DiagnosticHelper {
```

Comment:
@saloponov it is not neccessary to create separate class just for this method. Just simple method in the DacExtensionDefaultAttributeFix is enough:
```
private static bool IsBoundField(Diagnostic diagnostic)
{
	diagnostic.ThrowOnNull(nameof(diagnostic));
	return diagnostic.Properties.TryGetValue(DiagnosticProperty.IsBoundField, out string boundFlag)
				? boundFlag == bool.TrueString
			        :  false;
}
```
And it is used like a normal method